fe/fe-core/src/main/java/com/starrocks/system/SystemInfoService.java

```diff
@@ -374,8 +374,16 @@ public Backend getBackendWithBePort(String host, int bePort) {
                 LOG.info("faild to get address by fqdn {}", e.getMessage());
```

Comment:
better to add UT for getBackendWithBePort and mock idToBackend to cover ip or FQDN set or not be select logical.